compatibility_server/pip_checker.py

```diff
@@ -87,6 +87,7 @@ def __init__(self,
         self.command_string = command_string
 
 
+# TODO: deprecate the following:
```

Comment:
Why?

Reply:
This will be replaced by the `PackageStatus` enum introduced in this PR in a future PR.